Hyper/Collisions/Projectile.cs

```diff
@@ -0,0 +1,57 @@
+﻿using BepuPhysics;
+using BepuPhysics.Collidables;
+using BepuUtilities.Memory;
+using Hyper.Collisions.Bepu;
+using Hyper.Meshes;
+
+namespace Hyper.Collisions;
+internal class Projectile
+{
+    public BodyHandle Body { get; private set; }
+    private TypedIndex _shape;
+    private float _lifeTime;
+    private bool _disposed;
+    public bool IsDead { get; private set; }
+    public ProjectileMesh Mesh { get => _mesh; }
+
+    private readonly ProjectileMesh _mesh;
+
+    private Projectile(ProjectileMesh mesh, float lifeTime)
+    {
+        _mesh = mesh;
+        _lifeTime = lifeTime;
+    }
+
+    public static Projectile CreateStandardProjectile(Simulation simulation, CollidableProperty<SimulationProperties> properties,
+        in RigidPose initialPose, in BodyVelocity initialVelocity)
+    {
+        var projectileShape = new Box(1, 1, 1);
+
+        ProjectileMesh mesh = new ProjectileMesh(1);
```

Comment:
That complicates the mechanics for the projectile a bit, because we need to align the shape to the initial velocity vector when throwing. I'll have to look into that